.travis.yml

```diff
@@ -1,33 +1,42 @@
+group: travis_latest
+sudo: required
 language: python
+cache: pip
 
-os:
-    - linux
+python:
+  - '2.7'
+  - '3.6'
 
 branches:
   only:
-    - master
     - dev
+    - master
+
+env:
+  - ROPPER=''
+  - ROPPER='ropper'
 
 addons:
   apt:
     packages:
-      - python3
-      - gcc
-      - g++
-      - pkg-config
       - gdb
+      - python2.7-dbg
+      - python3-all-dbg
       - python3-pip
-      - pylint
-      - git
-      - cmake
-      - libglib2.0-dev
 
 before_script:
-  - pip3 install --user --upgrade ropper retdec-python keystone-engine capstone unicorn
+  - /usr/bin/python3 -m pip install --user capstone keystone-engine ${ROPPER}
+  - if [[ ${TRAVIS_PYTHON_VERSION} == '2.7' ]]; then
+      /usr/bin/python3 -m pip install --user unicorn;
+    else
```

Comment:
Why is sudo dependent on the python version?

Reply:
It is a mystery for me...  But my gut says that when "python" is not python2 then something in the unicorn build process generates a false message.  Unicorn is Python 3 compatible but something in its build process is not.